gfx/vr/FxRWindowManager.cpp

```diff
@@ -613,15 +613,14 @@ void FxRWindowManager::OnFullScreenChange(uint64_t aOuterWindowID,
   if (IsFxRWindow(aOuterWindowID)) {
     mIsInFullscreen = aIsFullScreen;
 
-    vr::VROverlayError overlayError;
     if (aIsFullScreen) {
       // Create the transport controls overlay
-      EnsureTransportControls();
-      overlayError = ChangeProjectionMode(VIDEO_PROJECTION_360);
+		EnsureTransportControls();
     } else {
       // Close the transport controls overlay
       HideTransportControls();
-      overlayError = ChangeProjectionMode(VIDEO_PROJECTION_2D);
+      vr::VROverlayError overlayError = ChangeProjectionMode(VIDEO_PROJECTION_2D);
+	  MOZ_ASSERT(overlayError == vr::VROverlayError_None);
     }
 
     MOZ_ASSERT(overlayError == vr::VROverlayError_None);
```

Comment:
this should have a build failure on a debug build. please remove this line

Reply:
Line 623?

Reply:
not 623, but 627 (since the scope of overlayError is now inside that else block)
MOZ_ASSERT should only be defined in debug, not release, which is why you wouldn't see that compile failure

Reply:
Ah, cool, got it. thanks